### PR TITLE
ariworkd added

### PR DESCRIPTION
**Pull request by StackAurora**

### CODE REVIEW

File: app/(main)/artwork/[slug]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
+"use client";
```
Comment (CodeMaster1022) [inline app/(main)/artwork/[slug]/page.tsx:1]: Good comment

File: app/(main)/artwork/[slug]/page.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+import { fetchServer, postServer } from "@/lib/net/fetch/fetch";
+import useNFTMint from "@/lib/web3/hook/nft/useNFTMint";
+import useToast from "@/lib/hooks/toast/useToast";
+import PromptAccordion from "@/lib/components/accordion/PromptAccordion";
+import ReactButton from "@/lib/components/button/ReactButton";
+import PrimaryButton from "@/lib/components/button/PrimaryButton";
+import { shortenAddress } from "@/lib/components/profile/profile-kit/ProfileHeader";
+
+type DetailedArtworkData = {
```
Comment (CodeMaster1022) [inline app/(main)/artwork/[slug]/page.tsx:14]: Sure.
Good part.

**Review by CodeMaster1022 — APPROVED** [review]
Perfect!

File: app/(main)/artwork/[slug]/page.tsx (hunk 2 above)
Comment (StackAurora) [reply app/(main)/artwork/[slug]/page.tsx:14]: Fixed

**Pull request merged**